projects/hmpps-auth-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/UserService.kt

```diff
@@ -17,11 +18,17 @@ import javax.naming.Name
 @Service
 class UserService(
     private val ldapTemplate: LdapTemplate,
-    private val auditUserService: AuditUserService
+    private val auditUserService: AuditUserService,
```

Comment:
Do you think we should move the existing calls to AuditUserService over to the new UserRepository for consistency?

Reply:
Yeah - much better - ive moved them and not using AuditRepository anymore. Done